packages/koa-joi-validate/test/joi16-17.test.js

```diff
@@ -0,0 +1,230 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const Joi16 = require('joi16');
+const Joi17 = require('joi17');
+const Koa = require('koa');
+const bodyParser = require('koa-bodyparser');
+const { describe, it } = require('mocha');
+const koaJoiValidate = require('..');
+
+chai.use(chaiHttp);
+chai.should();
+
+const policeMan = {
+  firstname: 'Lucas',
+  lastname: 'Hood',
+  age: 44,
+  job: 'Sheriff',
+  city: 'Banshee',
+  address: {
+    number: '35A',
+    street: 'Ford avenue'
+  }
+};
+const policeManController = (ctx) => {
+  ctx.status = 200;
+  ctx.body = ctx.request.body;
+};
+
+[Joi16, Joi17].forEach((Joi) => {
```

Comment:
There's a lot of shared logic between index.test.js and this one, can't we factorize it ? not absolutely necessary though